libraries/ms-drivers/test/test_bts_7200_current_sense.c

```diff
@@ -12,14 +12,57 @@
 #define TEST_CONFIG_PIN_I2C_SDA \
   { GPIO_PORT_B, 11 }
 
+// 1.6k resistor
+#define BTS7200_TEST_RESISTOR 1600
+
+// 1 mA * BTS7200_TEST_RESISTOR
+#define ADC_EXPECTED_OK_VOLTAGE 1600
+// See bts_7200_current_sense.h for an explanation of this value
+#define ADC_TEST_FAULT_VOLTAGE 3300
+
+// Return ADC_EXPECTED_OK_VOLTAGE by default
+#define ADC_DEFAULT_RETURNED_VOLTAGE_RAW ADC_EXPECTED_OK_VOLTAGE
+
+static uint16_t s_adc_measurement_0 = ADC_EXPECTED_OK_VOLTAGE;
+static uint16_t s_adc_measurement_1 = ADC_EXPECTED_OK_VOLTAGE;
+// To give different measurements between adc readings in bts_7200_get_measurement
+static bool s_adc_measurement_number = 0;
+
 static volatile uint16_t times_callback_called = 0;
 static void *received_context;
 
+static volatile uint16_t s_times_fault_callback_called = 0;
+static void *fault_received_context;
```

Comment:
All of these should be prefixed with `s_` (even though I realize 2 of them were preexisting)